### ISSUE TITLE
Feature Request: "Deadly Map Mods"

### ISSUE THREAD

**Issue by Rottenbeer** [briansd9/exile-diary#54]
Hey :)
Since you already have all the infos available, I would like to have a statistics page what shows me a ranking of the "most deadly" map mods to my build.
I guess it's easy to compare the mods on maps I died and search for similiarties.
It will be very helpful when leveling 90+ because I can avoid dangerous mods.
Thanks a lot!

**Comment by briansd9** [comment]
This is an interesting idea... will need some work though.

**Comment by btrisler** [comment]
I agree, I assume you're using the screenshot to get Level, IIQ, IIR, and Packsize.  To me it's the map mods which are killing me, not a particular map.  So it would be nice to have (at least in the export) which map mods were applied.  

I think this might be quite a bit more data than we're currently tracking.  Also given each map could have anywhere from 0 to 100 map mods, it's going to be hard to find a way to store that data without some kind of data structure.  

This would be a HUGE help.

Love your work
